src/sentry/tasks/reprocessing2.py

```diff
@@ -223,7 +227,9 @@ def finish_reprocessing(project_id, group_id):
         # While we migrated all associated models at the beginning of
         # reprocessing, there is still the "reprocessing" activity that we need
         # to transfer manually.
-        activity = Activity.objects.get(group_id=group_id)
+        # Any activities created during reprocessing (e.g. user clicks "assign" in an old browser tab)
+        # are ignored.
+        activity = Activity.objects.get(group_id=group_id, type=ActivityType.REPROCESS.value)
```

Comment:
I decided against moving multiple reprocessing activities to the new group, because that creates new problems: When those activities have different `newGroupIds`, what group do you copy to?

Instead, if this case every occurs, just let it crash and fix the bug at the source.